conanfile.py

```diff
@@ -93,7 +93,7 @@ def requirements(self):
             if not self.options.system_mesa:
                 self.requires("libxi/1.7.10@bincrafters/stable#0")
             if not self.options.system_qt:
-                self.requires("qt/5.14.1@bincrafters/stable#0")
+                self.requires("qt/5.15.0@{}#3b376b030c38da6378a2c2cea5f99418".format(self._orbit_channel))
```

Comment:
Any reason for that specific version? Maybe add a comment?

Reply:
I went this the latest one for having the latest version of Chromium. What should I write in the comment? If it's about "why that version change", I feel this should be part of the commit message. I could elaborate there if you prefer.